### ISSUE TITLE
New/Updated CFR date not syncing from AXIS to MOD

### ISSUE THREAD

**Issue by liseandtea** [bcgov/foi-flow#3683]
**Describe the bug in current situation**
After an applicant has paid a fee and a request comes off hold, and a new CFR has been sent, MOD is not updating with the new CFR due date. Instead, it is still pulling the original CFR due date from AXIS. This is shown in the images below where the original CFR due date was March 23, 2023, and not the new CFR due date of May 5, 2023. Because of this, the CFR is showing as 19 days overdue.

 ![New CFR Due Date.jpg](https://images.zenhubusercontent.com/633f19609b1468159fc7d8b5/435be4de-e36b-4ea7-ab8f-9c4282bc6378)

**Link bug to the User Story**

**Impact of this bug**
Analysts and Ministry coordinators will not have the correct CFR due date. 

**Chance of Occurring (high)**

**Pre Conditions: which Env, any pre-requesites or assumptions to execute steps?**

**Steps to Reproduce**
1. Go to 'any request that is on hold for fees, pay the fee, take the request off hold, and send a new CFR in AXIS'
2. RE-sync file to MOD
3. Check the "CFR due in XX days*

**CFR due date will show with a negative balance (overdue)**

**Expected behaviour**
CFR due date should adjust and calculate based on the most recent CFR sent. 

**Screenshots/ Visual Reference/ Source**
If applicable, add screenshots to help explain your problem. You an use screengrab.

**Comment by liseandtea** [comment]
About the potential bug we talked about this morning, it may not actually be a bug. What I think happened is: 1) the analyst put the payment into MOD, 2) took the request off hold, 3)  reassigned it back to the flex team, and 4) didn't wait long enough for the Flex team analyst to send out the new CFR. The CFR due date is now showing correctly. So I will create a request in AXIS, place it on hold for a few days, and try to recreate the bug and see what happens.

**Comment by liseandtea** [comment]
Tried to re-create this and can confirm it is not a bug therefore closing this ticket. 

Steps to reproduce the behavior:

1. I opened a request in AXIS 
2. I sent the CFR in AXIS on April 21, 2023, which had a due date of May 5, 2023
3. I synced the request to MOD - all dates showed correctly.
4. I put the request on hold in AXIS on April 26, 2023. The request remained open in MOD.
5. I took the request off hold in AXIS and sent a new CFR on April 27, 2023.
6. I resynced the request with MOD on April 27, 2023, and the CFR was updated to show a new due date of May 11, 2023. 

![image.png](https://images.zenhubusercontent.com/633f19609b1468159fc7d8b5/8a902e4a-6560-44b0-a68a-0c2249b34970)

![image.png](https://images.zenhubusercontent.com/633f19609b1468159fc7d8b5/1077223e-e79f-44db-b461-c76596ddb140)
